status/status.go

```diff
@@ -96,3 +130,16 @@ func printConn(e *status.Response) {
 	sort.Strings(a)
 	log.Print(strings.Join(a, "\n"))
 }
+
+func printJSON(all []se) {
+	b1 := new(bytes.Buffer)
+	e := json.NewEncoder(b1)
+	e.Encode(all)
+
+	b2 := new(bytes.Buffer)
+	json.Indent(b2, b1.Bytes(), "", "  ")
+
+	out := bufio.NewWriter(os.Stdout)
```

Comment:
Why don't you use `fmt.Print` here?

Reply:
os.Stdout is non-buffered by default. If you know/think you are going to be writing a lot to it, then it is better to wrap it in a bufio first.